testing/mod.ts

```diff
@@ -69,8 +69,29 @@ function print(txt: string, newline: boolean = true): void {
   Deno.stdout.writeSync(encoder.encode(`${txt}`));
 }
 
+declare global {
+  interface Window {
```

Comment:
Is it really necessary to declare this before hand?

Reply:
Without it TS complains that `__DENO_TEST_REGISTRY` doesn't exist on `Window`:
```
error TS2339: Property '__DENO_TEST_REGISTRY' does not exist on type 'Window'.

► file:///Users/biwanczuk/dev/deno_std/testing/mod.ts:90:23

90   candidates = window.__DENO_TEST_REGISTRY as TestDefinition[];
```

I can add `@ts-ignore` but this solution is more idiomatic

Reply:
Ok sounds good.